app/views/wpcc/your_details/new.html.erb

```diff
@@ -42,10 +39,7 @@
         <div class="details__field">
           <div class="details__field-label">
             <%= f.label :gender, t('wpcc.details.gender.label'), class: 'form__label-heading' %>
-            <button type="button" class="popup-tip__button" data-dough-popup-trigger>
-              <span aria-hidden="true">i</span>
-              <span class="visually-hidden"><%= t('wpcc.details.tooltip_show') %></span>
-            </button>
+            <%= render 'wpcc/shared/popup_tip_trigger' %>
```

Comment:
Yes. Note that if you call in the view as `render tooltips/show` then is a partial on app/views/wpcc/tooltips/_show.html.erb (note the underscore). Or you can call it `render template: 'tooltips/show'` and have app/views/wpcc/tooltips/show.html.erb as normal. It is up to you I think on what it is best.